src/main/java/net/dv8tion/jda/internal/entities/MemberImpl.java

```diff
@@ -116,6 +116,13 @@ public OffsetDateTime getTimeBoosted()
         return boostDate != 0 ? Helpers.toOffset(boostDate) : null;
     }
 
+    @Nullable
+    @Override
+    public OffsetDateTime getTimeUntilTimedOut()
```

Comment:
This sounds more like the relative time until the timeout starts

Reply:
suggestions are welcome, as I can't think of a better name.